constants/device.go

```diff
@@ -26,7 +25,7 @@ const (
 	DEFAULT_EDISON_IP     = "192.168.2.15"
 
 	DEFAULT_RASPBERRY_USERNAME  = "pi"
-	DEFAULT_RASPBERRY_PASSWORD  = "password"
+	DEFAULT_RASPBERRY_PASSWORD  = "raspberry"
```

Comment:
__[golint]__
* [golint] don't use ALL_CAPS in Go names; use CamelCase

<!-- Comment by [SideCI](https://sideci.com) -->